catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPostIngestPlugin.java

```diff
@@ -201,34 +245,18 @@ private void process(ContentItem contentItem, List<ContentItem> contentItems) {
 
         if (storeOriginalImage) {
           ContentItem originalImageContentItem =
-              createOriginalImage(
-                  contentItem.getId(), renderImageUsingOriginalDataModel(contentItem), metacard);
+              createOriginalImage(contentItem.getId(), renderedImage, metacard);
 
           contentItems.add(originalImageContentItem);
         }
       }
-    } catch (IOException | ParseException | NitfFormatException | RuntimeException e) {
+    } catch (IOException | NitfFormatException | RuntimeException e) {
       LOGGER.debug(e.getMessage(), e);
     }
   }
 
-  private BufferedImage renderImage(ContentItem contentItem)
```

Comment:
I think there was a reason why both `renderImage` and `renderImageUsingOriginalDataModel` were needed. I can't remember, but it might have had something to do with JPEG200 NITFS.

Reply:
I talked with @glenhein briefly about this(he wrote the original code) and he couldn't remember any reason for this. I think using the same data model is the correct way to go though and it has the benefit of being faster since we don't have to re-render anything. I will double check that there isn't an issue with JPEG2000 nitfs.